lascli/parser/models.py

```diff
@@ -138,6 +147,11 @@ def create_models_parser(subparsers):
     list_data_bundles_parser.add_argument('--next-token', '-n', type=str, default=None)
     list_data_bundles_parser.set_defaults(cmd=list_data_bundles)
 
+    list_all_data_bundles_parser = subparsers.add_parser('list-all-data-bundles')
+    list_all_data_bundles_parser.add_argument('--max-results', '-m', type=int, default=None)
+    list_all_data_bundles_parser.add_argument('--next-token', '-n', type=str, default=None)
```

Comment:
Do we want to support `--next-token` and `--max-results` when listing **all** data bundles?